cypress/e2e/ChatbotPage.cy.ts

```diff
@@ -0,0 +1,29 @@
+describe('Chatbot page', () => {
+  const username = 'user';
+  const password = 'pass';
+  beforeEach(() => {
+    cy.visit('/login');
+    cy.get('[data-cy="username-text-field"]').type(username);
+    cy.get('[data-cy="password-text-field"]').type(password);
+    cy.get('[data-cy="sign-in-button"]').click();
+  });
+  it('Shows chatbot page when logged in', () => {
+    cy.url().should('include', '/');
+    cy.get('[data-cy="chatbot"]').should('exist');
+  });
+
+  it('Send message to chatbot and get response', () => {
+    cy.get('[data-cy="chatbot"]').should('exist');
+    cy.get('[data-cy="chatbot-input-field"]').type('Hello');
+    cy.get('[data-cy="chatbot-send-button"]').click();
+    cy.get('[data-cy="user-input-message"]').should('exist');
+    cy.wait(3000);
```

Comment:
This wait is unnecessary, see:
- https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting
- https://docs.cypress.io/guides/guides/network-requests#Waiting